docs/_docs/themes.md

```diff
@@ -1,580 +1,176 @@
 
-```liquid
-{% raw %}
-{{ site.baseurl }}{% post_url /subdir/2010-07-21-name-of-post %}
-{% endraw %}
+```sh
+jekyll new-theme my-awesome-theme
+             create /path/to/my-awesome-theme/_layouts
+             create /path/to/my-awesome-theme/_includes
+             create /path/to/my-awesome-theme/_sass
+             create /path/to/my-awesome-theme/_layouts/page.html
+             create /path/to/my-awesome-theme/_layouts/post.html
+             create /path/to/my-awesome-theme/_layouts/default.html
+             create /path/to/my-awesome-theme/Gemfile
+             create /path/to/my-awesome-theme/my-awesome-theme.gemspec
+             create /path/to/my-awesome-theme/README.md
+             create /path/to/my-awesome-theme/LICENSE.txt
+         initialize /path/to/my-awesome-theme/.git
+             create /path/to/my-awesome-theme/.gitignore
+Your new Jekyll theme, my-awesome-theme, is ready for you in /path/to/my-awesome-theme!
+For help getting started, read /path/to/my-awesome-theme/README.md.
 ```
 
-There is no need to include the file extension when using the `post_url` tag.
+Add your template files in the corresponding folders. Then complete the `.gemspec` and the README files according to your needs.
 
-You can also use this tag to create a link to a post in Markdown as follows:
+### Layouts and includes
 
-```liquid
-{% raw %}
-[Name of Link]({{ site.baseurl }}{% post_url 2010-07-21-name-of-post %})
-{% endraw %}
-```
+Theme layouts and includes work just like they work in any Jekyll site. Place layouts in your theme's `/_layouts` folder, and place includes in your themes `/_includes` folder.
+
+For example, if your theme has a `/_layouts/page.html` file, and a page has `layout: page` in its YAML front matter, Jekyll will first look to the site's `_layouts` folder for a the `page` layout, and if none exists, will use your theme's `page` layout.
+
+### Assets
+
+Any file in `/assets` will be copied over to the user's site upon build unless they have a file with the same relative path. You can ship any kind of asset here: SCSS, an image, a webfont, etc. These files behave just like pages and static files in Jekyll: if the file has [YAML front matter](../docs/frontmatter/) at the top, then it will be rendered. If it does not have YAML front matter, it will simply be copied over into the resulting site. This allows theme creators to ship a default `/assets/styles.scss` file which their layouts can depend on as `/assets/styles.css`.
+
+All files in `/assets` will be output into the compiled site in the `/assets` folder just as you'd expect from using Jekyll on your sites.
+
+### Stylesheets
+
+Your theme's stylesheets should be placed in your theme's `/_sass` folder, again, just as you would when authoring a Jekyll site. Your theme's styles can be included in the user's stylesheet using the `@import` directive.
```

Comment:
Shouldn't we show an example here? Should we imply a convention like prefixing the gem with *jekyll-theme-*?

Your theme's stylesheets should be placed in your theme's `/_sass` folder, again, just as you would when authoring a Jekyll site.

```sass
  _sass
├── jekyll-theme-my-awesome-theme.scss
```
Your theme's styles can be included in the user's stylesheet using the `@import` directive.

`@import "{{ site.theme }}";`